upload/views/empty_upload.py

```diff
@@ -67,6 +71,15 @@ class EmptyUploadView(CreateAPIView, GetterMixin):
     ]
 
     def post(self, request, *args, **kwargs):
+        serializer = EmptyUploadSerializer(data=request.data)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        data = serializer.validated_data
+        if "should_force" in data:
```

Comment:
This is VERY nit, but doesn't `should_force = data.get("should_force", False)` accomplishes the same?